code/modules/projectiles/ammo_datums.dm

```diff
@@ -871,13 +871,21 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	damage = 75
 	penetration = 35
 
+/datum/ammo/bullet/sniper/railgun
+	name = "HVP"
+	hud_state = "HVP"
```

Comment:
Time to fix some strings again:
`name `shouldn't be shortened to `HVP`, should it go to `high velocity projectile` instead?
`hud_state`, for consistency should be `sniper_railgun` or `sniper_hvp` instead of `HVP`?